A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_Init.js

```diff
@@ -0,0 +1,377 @@
+class CDUAocInit {
+    static ShowPage(mcdu) {
+        mcdu.clearDisplay();
+        mcdu.page.Current = mcdu.page.AOCInit;
+        mcdu.activeSystem = 'ATSU';
+
+        let fltNbr = '_______[color]red';
+        let originIcao = '____[color]red';
+        let destinationIcao = '____[color]red';
+        let ete = "____[color]red";
+        let fob = `{small}---.-{end}[color]white`;
+
+        const currentFob = formatWeight(SimVar.GetSimVarValue("FUEL TOTAL QUANTITY WEIGHT", "kg"));
+
+        function formatWeight(value) {
+            return (value / 1000).toFixed(1);
+        }
+
+        if (mcdu.simbrief["originIcao"]) {
+            originIcao = `${mcdu.simbrief["originIcao"]}[color]blue`;
+        }
+        if (mcdu.simbrief["destinationIcao"]) {
+            destinationIcao = `${mcdu.simbrief["destinationIcao"]}[color]blue`;
+        }
+        if (mcdu.simbrief["icao_airline"] || mcdu.simbrief["flight_number"]) {
+            fltNbr = `${mcdu.simbrief.icao_airline}${mcdu.simbrief.flight_number}[color]green`;
+        }
+        if (mcdu.simbrief["ete"]) {
+            ete = `${mcdu.simbrief.ete}[color]blue`;
+        }
+        if (currentFob) {
+            fob = `{small}${currentFob}{end}[color]green`;
+        }
+
+        const display = [
+            ["INIT/REVIEW", "1", "2", "AOC"],
+            ["FMC FLT NO", "GMT"],
+            [fltNbr, "2337[color]green"],
+            ["DEP"],
+            [originIcao],
+            ["DEST"],
+            [destinationIcao, "CREW DETAILS>"],
+            ["FOB"],
+            ["   " + fob],
+            ["ETE"],
+            [ete, "INIT DATA REQ*[color]blue"],
+            ["", "ADVISORY"],
+            ["<AOC MENU"]
+        ];
+        mcdu.setTemplate(display);
+
+        mcdu.rightInputDelay[4] = () => {
+            return mcdu.getDelayBasic();
+        };
+        mcdu.onRightInput[4] = () => {
+            CDUAocInit.getSimbrief(mcdu);
+        };
+
+        mcdu.leftInputDelay[5] = () => {
+            return mcdu.getDelaySwitchPage();
+        };
+        mcdu.onLeftInput[5] = () => {
+            CDUAocMenu.ShowPage(mcdu);
+        };
+
+        mcdu.onNextPage = () => {
+            CDUAocInit.ShowPage2(mcdu);
+        };
+    }
+
+    static ShowPage2(mcdu) {
+        mcdu.clearDisplay();
+        mcdu.page.Current = mcdu.page.AOCInit2;
+        mcdu.activeSystem = 'ATSU';
+
+        const currentFob = formatWeight(SimVar.GetSimVarValue("FUEL TOTAL QUANTITY WEIGHT", "kg"));
+
+        function formatWeight(value) {
+            return (value / 1000).toFixed(1);
+        }
+
+        function formatTime(timestamp) {
+            var date = new Date(+timestamp * 1000);
+            return `${date.getUTCHours().toString().padStart(2, "0")}${date.getUTCMinutes().toString().padEnd(2, "0")}`;
```

Comment:
this method may already exist... I'm not too sure if not, extract it to FMCMainDisplay.js so others can use it?
https://github.com/flybywiresim/a32nx/blob/58ecd2b5844d5f4263f0c86b87c5e3652819df82/A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js#L3295-L3345

Reply:
I thinks it fits the case. Ill use those methods, thanks!